test/lager_test_function_transform.erl

```diff
@@ -122,10 +124,11 @@ transform_function_test_() ->
           ?assertEqual(1, lager_test_backend:count()),
           {_Level, _Time, _Message, Metadata} = lager_test_backend:pop(),
           Function = proplists:get_value(returns_dynamic_emit, Metadata),
-          ?assert(Function() < Function()),
-          ?assert(Function() < Function()),
-          ?assert(Function() < Function()),
-          ?assert(Function() < Function()),
+                ct:pal("Function:~p~n", [Function]),
```

Comment:
Why did all of this get changed? To emit log messages in eunit, you need to use the `?debugFmt()` macro not ct:pal :)

Reply:
I am sorry, this is just my test code, I will remove it.